UPGRADING.rst

```diff
@@ -10,6 +10,21 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#4011 Integrate monitoring SNS topic with Slack
+===============================================
+
+Operator
+~~~~~~~~
+
+Complete the following steps shortly before pushing the merge commit to the
+GitLab instance for any main deployment. Set up AWS Chatbot Slack client as
+`described <./README.md#313-aws-chatbot-integration-with-slack>`_ in the README.
+This has already been done for the ``platform-hca-dev`` account but is still
+needed for ``platform-anvil-dev`` and ``platform-hca-prod``.
+
+Manually deploy the ``shared`` component in that deployment.
```

Comment:
It is not clear what "that deployment" is referring to and whether `dev.shared` needs to be deployed.

```suggestion
Before pushing a merge commit with these changes to a GitLab instance, `set up AWS Chatbot`<./README.md#313-aws-chatbot-integration-with-slack>`_ in the AWS account hosting that instance. AWS Chatbot has already been set up in the ``platform-hca-dev`` account. Once AWS Chatbot is set up, manually deploy the ``shared`` component of the main deployment collocated with the GitLab instance you will be pushing to.
```